lib/menu.rb

```diff
@@ -0,0 +1,17 @@
+class Menu
+
+  attr_reader:list
```

Comment:
Lint/SpaceBeforeFirstArg: Put space between the method name and the first argument.